content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -1,130 +1,106 @@
+Terraform variables refer to [input variables](/docs/language/values/variables.html) that define parameters without hardcoding them into the configuration. For example, you could create variables that let users specify the number and type of Amazon Web Services EC2 instances they want to provision with a Terraform module. If a required input variable is missing, Terraform plans in the workspace will fail and print an explanation in the log.
 
-To edit a variable, click one of its text fields or its pencil (edit) icon to reveal the editing controls. Make any desired changes to the variable's name, value, description, and settings, then click the "Save Variable" button.
+Terraform Cloud passes input variables to Terraform by writing a `terraform.tfvars` file, meaning that Terraform Cloud overwrites any `terraform.tfvars` file you check into version control. You should not check in `terraform.tfvars` files even when running Terraform solely on the command line.
 
-To add a variable, click the "+ Add Variable" button, enter a name, value, optional description, and save.
+You must define input variables manually; Terraform Cloud does not automatically discover variable names from Terraform code. However, workspaces that use Terraform 0.10.0 or later allow you to commit any number of [`*.auto.tfvars` files](/docs/language/values/variables.html#variable-files) to provide default Terraform input variables. If any variables from these files have the same key as existing variables in the Terraform Cloud workspace, the variables applied to the workspace overwrite variables from the files.
 
-To delete a variable, click its "🗑" (trash can) icon, then confirm your decision in the dialog box that appears.
+### Environment Variables
 
-You can edit one variable at a time, and must save or cancel your current edits before editing other variables in the list.
+Terraform Cloud performs Terraform runs on disposable Linux worker VMs using a POSIX-compatible shell. Before running Terraform operations, Terraform Cloud uses the `export` command to populate the shell with environment variables.
 
-![Screenshot: A variable being edited](./images/vars-edit.png)
+Environment variables can store provider credentials and other data. Refer to your provider's Terraform Registry documentation for a full list of supported shell environment variables (e.g., authentication variables for [AWS](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#environment-variables), [Google Cloud Platform](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/getting_started#adding-credentials), and [Azure](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#argument-reference)). Environment variables can also [modify Terraform's behavior](/docs/cli/config/environment-variables.html). For example, `TF_LOG` enables detailed logs for debugging.
 
-### Multi-line Values
+#### Parallelism
```

Comment:
This section seems a little bit in the weeds compared to the rest of the content on this page. Not sure if there's anywhere better to put it though.

Reply:
yeah this is a relic of the old content. I wanted to combine things - there used to be a "Special Environment Variables" section, which I thought was a bit weird. I don't love this, but....I think I'll keep it for now. But I agree - it is definitely very specific given the rest of the content on the page.